packages/ui/components/form-core/types/validate/ValidateMixinTypes.ts

```diff
@@ -9,9 +9,9 @@ import { FormControlHost } from '../FormControlMixinTypes.js';
 import { SyncUpdatableHost } from '../utils/SyncUpdatableMixinTypes.js';
 import { Validator } from '../../src/validate/Validator.js';
 
-type FeedbackMessage = {
+export type FeedbackMessage = {
   message: string | Node;
-  type: string;
+  type: ValidationType;
```

Comment:
Someone extending lion should still be able to specify a type beyond error/warning/info/success, so I'm not 100% sure about this...